trace-mini-agent/src/config.rs

```diff
@@ -41,6 +42,10 @@ impl Config {
             anyhow::anyhow!("Unable to identify environment. Shutting down Mini Agent.")
         })?;
 
+        let dd_dogstatsd_port: u16 = env::var("DD_DOGSTATSD_PORT")
+            .ok()
+            .and_then(|port| port.parse::<u16>().ok())
+            .unwrap_or(8125);
```

Comment:
magic number

Reply:
Added an environment variable `DEFAULT_DOGSTATSD_PORT` for this as well as an `AGENT_HOST` environment variable.